lib/util/PDF/create_pdf.dart

```diff
@@ -0,0 +1,205 @@
+            textDirection: pw.TextDirection.rtl,
+            child: pw.Column(children: [
+              pw.Container(
+                width: pageFormat.availableWidth,
+                child: pw.Align(
+                  alignment: pw
+                      .Alignment.centerRight, // Align the content to the right
+                  child: pw.Directionality(
+                    textDirection:
+                        pw.TextDirection.rtl, // Set text direction to RTL
+                    child: pw.Text(
+                      titles[i],
+                      style: pw.TextStyle(
+                          fontSize: 26,
+                          fontWeight: pw.FontWeight.bold,
+                          font: ttf),
+                    ),
+                  ),
+                ),
+              ),
+              pw.SizedBox(height: 15),
+              pw.Container(
+                width: pageFormat.availableWidth,
+                child: pw.Align(
+                  alignment: pw
+                      .Alignment.centerRight, // Align the content to the right
+                  child: pw.Directionality(
+                    textDirection:
+                        pw.TextDirection.rtl, // Set text direction to RTL
+                    child: pw.Text(
+                      subTitles[i],
+                      style: pw.TextStyle(
+                          fontSize: 20,
+                          fontWeight: pw.FontWeight.bold,
+                          font: ttf),
+                    ),
+                  ),
+                ),
+              ),
+              pw.SizedBox(height: 15),
+              pw.Container(
+                color: const PdfColor.fromInt(0xfaf6fd),
+                child: pw.Table(
+                  border: pw.TableBorder.all(),
+                  children: [
+                    for (var entry in data[i].asMap().entries)
+                      pw.TableRow(
+                        children: [
+                          pw.Padding(
+                              padding: const pw.EdgeInsets.only(right: 5),
+                              child: pw.Directionality(
+                                  textDirection: pw.TextDirection.rtl,
+                                  child: pw.Text(
+                                      '${entry.key + 1}. ${entry.value}',
+                                      style:
+                                          pw.TextStyle(fontSize: 20, font: ttf),
+                                      textAlign: pw.TextAlign.right))),
+                        ],
+                      ),
+                  ],
+                ),
+              ),
+            ])),
+      ),
+    );
+  }
+
+  // Add space before footer
+  widgets.add(
+    pw.Expanded(
+      child: pw.SizedBox(),
+    ),
+  );
+
+  // Add the footer content to the PDF
+  widgets
+      .add(pw.Column(crossAxisAlignment: pw.CrossAxisAlignment.end, children: [
+    pw.Directionality(
+      textDirection: pw.TextDirection.rtl,
```

Comment:
We plan to make the app multi-lingual. wouldn't this be a problem when creating english based plans?

Reply:
Without appropriate changes, yes. But I think that's ok because we haven't even thought about how multi-lingualism will be implemented. Once we know that we can change that to a emun or w/e that decides rather to use TextDirection.rtl or TextDirection.ltr. 

In short, the needed change is minor enough and requires knowledge about the implementation such that IMO it shouldn't part of production at this point.
In the same vein, I think it's better to have another pr that deals with multi-lingual implementations throughout the app instead of doing it only for this feature.

Reply:
I'm ok with leaving this as it is but TBH you should write the code to be compatible with any multi-lingual lib we will choose. the utility layer shouldn't even know you are multi-lingual. it should get the direction data from its caller